workspaces/sonarqube/package.json

```diff
@@ -37,13 +43,15 @@
     "@backstage/repo-tools": "^0.9.4",
     "@changesets/cli": "^2.27.1",
     "@spotify/prettier-config": "^15.0.0",
+    "concurrently": "^8.0.0",
     "node-gyp": "^10.0.0",
     "prettier": "^2.3.2",
     "typescript": "~5.3.0"
   },
   "resolutions": {
     "@types/react": "^18",
-    "@types/react-dom": "^18"
+    "@types/react-dom": "^18",
+    "csstype@^3.0.2": "3.0.9"
```

Comment:
Can you add `"@microsoft/api-extractor": "7.36.4",` as this will be needed for the API Reports as a temporary fix to an upstream issue. Then you can delete the current API reports you have checked in in this PR and regenerate them.